spec/integration/api/v1/ingredients_spec.rb

```diff
@@ -51,8 +52,175 @@
         run_test!
       end
 
-      response "401", "with invalid user token" do
-        let(:user_token) { "invalid" }
+      response '401', 'user unauthorized' do
+        let(:user_token) { 'invalid' }
+
+        run_test!
+      end
+    end
+  end
+
+  path '/ingredients' do
+    parameter name: :user_email, in: :query, type: :string
+    parameter name: :user_token, in: :query, type: :string
+
+    get 'Retrieves Ingredients' do
+      description 'Retrieves all the ingredients'
+      produces 'application/json'
+
+      let(:collection_count) { 5 }
+      before do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/ScatteredSetup: Do not define multiple before hooks in the same example group (also defined on line 26).